deepchecks/checks/performance/performance_report.py

```diff
@@ -169,7 +176,7 @@ def condition(check_result: pd.DataFrame) -> ConditionResult:
                     test_scores_dict = dict(zip(test_scores_class['Metric'], test_scores_class['Value']))
                     train_scores_dict = dict(zip(train_scores_class['Metric'], train_scores_class['Value']))
                     # Calculate percentage of change from train to test
-                    diff = {score_name: ((score - test_scores_dict[score_name]) / score)
+                    diff = {score_name: _ratio_calc(score, test_scores_dict[score_name])
                             for score_name, score in train_scores_dict.items()}
                     failed_scores = [k for k, v in diff.items() if abs(v) > threshold]
```

Comment:
Reading the description I would assume only score_train > score_test can raise the condition, but here I see abs()